custom_components/hubitat/climate.py

```diff
@@ -236,6 +236,13 @@ def temperature_unit(self) -> str:
             return TEMP_FAHRENHEIT
         return TEMP_CELSIUS
 
+    @property
+    def precision(self) -> Optional[float]:
+        """Return current temperature precision when Farenheit and thermostat reports decimals."""
+        if self.temperature_unit == TEMP_FAHRENHEIT and "." in str(self.current_temperature):
```

Comment:
The condition shouldn't have the value check, just a unit check. If Hubitat can send floats for Fahrenheit values, the climate entity should always report the precision as `PRECISION_TENTHS`.

Reply:
Hey Jason, thanks for taking a look at this! My thinking with this approach was that for other thermostats, the tenths wouldn't be shown when not applicable. For instance, here's the other thermostat in my house - its a "real" thermostat, not virtual. This device will never report a tenths reading, so I like that it doesn't show "66.**0**".  I can live with it though, if it means I get tenths on my virtual thermostat! Any thoughts? 

![image](https://user-images.githubusercontent.com/73209289/100815567-406ef000-3412-11eb-8d13-8a672ff2a008.png)